### PR TITLE
Update the Hardhat Console guide

### PR DESCRIPTION
**Pull request by alcuadrado**

### CODE REVIEW

File: docs/src/content/hardhat-runner/docs/guides/hardhat-console.md [hunk 1]
```diff
@@ -41,33 +43,35 @@ And the initialized `ethers` object if you're using the `hardhat-ethers` plugin:
...
 ```
 
-And the `artifacts` object if you're using the `hardhat-truffle5` plugin, and so on.
+Anything that has been injected into the [Hardhat Runtime Environment] will be magically available in the global scope.
 
-Anything that has been injected into the [Hardhat Runtime Environment] will be magically available in the global scope. Alternatively, if you're the more explicit kind of developer, you can instead require the HRE explicitly and get autocomplete:
+Alternatively, if you're the more explicit kind of developer, you can instead require the HRE explicitly:
 
 ```
 > const hardhat = require("hardhat")
```
Comment (fvictorio) [inline docs/src/content/hardhat-runner/docs/guides/hardhat-console.md:51]: Can we change this to use `hre` instead, for consistency with the rest of the docs?

File: docs/src/content/hardhat-runner/docs/guides/hardhat-console.md [hunk 2]
```diff
@@ -41,33 +43,35 @@ And the initialized `ethers` object if you're using the `hardhat-ethers` plugin:
...
+
+You will also notice that the console has the handy history feature you expect out of most interactive terminals, including across different sessions. Try it by pressing the up arrow key. The Hardhat console is just an instance of a Node.js console, so anything you use in Node.js will also work.
 
 ### Asynchronous operations and top-level await
 
-Interacting with the Ethereum network, and therefore with your smart contracts, are asynchronous operations. Therefore most APIs and libraries use JavaScript's `Promise` for returning values.
+Interacting with the Ethereum network, and therefore with your smart contracts, are asynchronous operations. Therefore, most APIs and libraries use JavaScript's `Promise` for returning values.
 
-To make things easier, Hardhat's console supports `await` top-level await (i.e. `console.log(await web3.eth.getBalance()`). To use this feature, you need to be using Node 10 or higher.
+To make things easier, Hardhat's console supports `await` top-level await (i.e. `console.log(await ethers.getSigners()`).
```
Comment (fvictorio) [inline docs/src/content/hardhat-runner/docs/guides/hardhat-console.md:74]:
```suggestion
To make things easier, Hardhat's console supports top-level `await` (i.e. `console.log(await ethers.getSigners()`).
```

Comment (fvictorio) [reply docs/src/content/hardhat-runner/docs/guides/hardhat-console.md:74]: Or maybe "top-level `await` statements"

File: docs/src/content/hardhat-runner/docs/guides/hardhat-console.md (hunk 1 above)
Comment (alcuadrado) [reply docs/src/content/hardhat-runner/docs/guides/hardhat-console.md:51]: yeah, makes sense

File: docs/src/content/hardhat-runner/docs/guides/hardhat-console.md (hunk 2 above)
Comment (alcuadrado) [reply docs/src/content/hardhat-runner/docs/guides/hardhat-console.md:74]: thanks

**Pull request merged**